libs/components/phone-field/src/lib/modules/phone-field/phone-field.component.ts

```diff
@@ -175,71 +175,81 @@ export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
+  public set selectedCountry(newCountry: SkyPhoneFieldCountry | undefined) {
     if (
       newCountry &&
-      (!this._selectedCountry || this._selectedCountry.iso2 !== newCountry.iso2)
+      (!this.#_selectedCountry ||
+        this.#_selectedCountry.iso2 !== newCountry.iso2)
     ) {
-      this._selectedCountry = this.countries.find(
+      this.#_selectedCountry = this.countries.find(
         (country) => country.iso2 === newCountry.iso2
       );
 
-      if (!this._selectedCountry.exampleNumber) {
-        const numberObj = this.phoneUtils.getExampleNumberForType(
+      if (this.#_selectedCountry && !this.#_selectedCountry.exampleNumber) {
+        const numberObj = this.#phoneUtils.getExampleNumberForType(
           newCountry.iso2,
           PhoneNumberType.FIXED_LINE
         );
-        this._selectedCountry.exampleNumber = this.phoneUtils.format(
+        this.#_selectedCountry.exampleNumber = this.#phoneUtils.format(
           numberObj,
           PhoneNumberFormat.NATIONAL
         );
       }
 
-      this.selectedCountryChange.emit(this._selectedCountry);
+      this.selectedCountryChange.emit(this.#_selectedCountry);
     }
   }
 
-  public get selectedCountry(): SkyPhoneFieldCountry {
-    return this._selectedCountry;
+  public get selectedCountry(): SkyPhoneFieldCountry | undefined {
+    return this.#_selectedCountry;
   }
 
   @ViewChild('inputTemplateRef', {
     read: TemplateRef,
     static: true,
   })
-  private inputTemplateRef: TemplateRef<unknown>;
+  public inputTemplateRef: TemplateRef<unknown> | undefined;
 
   @ViewChild('countryBtnTemplateRef', {
     read: TemplateRef,
     static: true,
   })
-  private countryBtnTemplateRef: TemplateRef<unknown>;
+  public countryBtnTemplateRef: TemplateRef<unknown> | undefined;
 
   @ViewChild('buttonsInsetTemplateRef', {
     read: TemplateRef,
     static: true,
   })
-  private buttonsInsetTemplateRef: TemplateRef<unknown>;
+  public buttonsInsetTemplateRef: TemplateRef<unknown> | undefined;
 
-  private defaultCountryData: SkyPhoneFieldCountry;
+  #defaultCountryData: SkyPhoneFieldCountry | undefined;
 
-  private phoneInputAnimationTriggered = false;
+  #phoneInputAnimationTriggered = false;
 
-  private phoneUtils = PhoneNumberUtil.getInstance();
+  #phoneUtils = PhoneNumberUtil.getInstance();
 
-  private longestDialCodeLength = 0;
+  #longestDialCodeLength = 0;
 
-  private _defaultCountry: string;
+  #_defaultCountry: string | undefined;
 
-  private _selectedCountry: SkyPhoneFieldCountry;
+  #_selectedCountry: SkyPhoneFieldCountry | undefined;
+
+  #countrySearchFormControl = new UntypedFormControl();
+
+  #formBuilder: UntypedFormBuilder;
+  #adapterService: SkyPhoneFieldAdapterService;
+  #changeDetector: ChangeDetectorRef;
 
   constructor(
-    private formBuilder: UntypedFormBuilder,
-    private adapterService: SkyPhoneFieldAdapterService,
-    private changeDetector: ChangeDetectorRef,
+    formBuilder: UntypedFormBuilder,
```

Comment:
In the spirit of strict mode, could you try to convert these `Untyped` form types into their "typed" versions?